src/ansys/geometry/core/designer/component.py

```diff
@@ -67,7 +67,11 @@
 from ansys.geometry.core.math.matrix import Matrix44
 from ansys.geometry.core.math.point import Point3D
 from ansys.geometry.core.math.vector import UnitVector3D, Vector3D
-from ansys.geometry.core.misc.checks import check_pint_unit_compatibility, ensure_design_is_active
+from ansys.geometry.core.misc.checks import (
```

Comment:
suprised that it didn't complain about it during build and test.

Reply:
Not sure you ran it properly then 🤫